pylzy/lzy/api/v1/startup.py

```diff
@@ -112,7 +112,7 @@ def process_execution(
     try:
         res = op(*args, **kwargs)
     except Exception as e:
-        logger.error(f"Execution completed with error {e} in {time.time() - start}")
+        logger.error(f"Execution completed with error `{e}` in {time.time() - start}")
```

Comment:
No f-formatting in logging, pls

Reply:
Also - this formatting is everywhere. Imho, it will be strange to have one different log. Let's refactor this in different PR, if this is important.